SECP-service/src/main/resources/assets/app/index.html

```diff
@@ -59,6 +64,8 @@
         <script src="lib/sweetalert2/dist/sweetalert2.js"></script>
         <script src="lib/select2/dist/js/select2.js"></script>
         <script src="lib/moment/moment.js"></script>
+        <script src="lib/bootstrap-css/js/bootstrap.min.js"></script>
```

Comment:
I guess we just didn't have the proper bootstrap then

Reply:
yep we forgot to add the bootstrap javascript file.